### PR TITLE
add dora

### PR DESCRIPTION
**Pull request by wuallenwu**
adding dora artifacts to isaac lab!

### CODE REVIEW

File: source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/dora.py [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Configuration for Stompy."""
+
+import omni.isaac.lab.sim as sim_utils
+from omni.isaac.lab.actuators import ImplicitActuatorCfg
+from omni.isaac.lab.assets.articulation import ArticulationCfg
+
+STOMPY_CFG = ArticulationCfg(
```
Comment (budzianowski) [inline source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/dora.py:12]: please update naming in all files

File: source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/dora.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""Configuration for Stompy."""
```
Comment (budzianowski) [inline source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/dora.py:6]: update

**Review by budzianowski — APPROVED** [review]
lgtm

File: source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/dora.py (hunk 1 above)
Comment (wuallenwu) [reply source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/dora.py]: fixed

File: source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/dora.py (hunk 2 above)
Comment (wuallenwu) [reply source/extensions/omni.isaac.lab_assets/omni/isaac/lab_assets/dora.py]: fixed

**Pull request merged**